tutorial/westeros/westeros.ipynb

```diff
@@ -0,0 +1,1268 @@
0fPjwdYVtly1btufFF1+8esmSJQu6dOny3vTp0/cFmDBhQsW11167bP78+X9pzjobuodrdXX1nuedd17t4sWLF+69995bfvjDH5bvyGtr6oSmrwFjC8bbA58AOpLchu7uHVm5mVlDKi7/TalLyFX1908udQklN3z48A3jx4/vuHbt2rL27dvHYYcdtv6RRx7p8Pjjj+/94x//eNl11123tW2PHj02Dhs27B2AwYMHb6iurm6/Zs2aNuvWrWtz8sknrwf40pe+tObBBx/s0sQ6P3AP1zFjxrzRvXv3TZ/5zGfeBjj77LPX3HDDDR8CVm3va2sqXNtFROHFjh+NiDXAGkkdt3elZmZm7du3j549e2686aabug0dOnT9oEGD3nnggQf2Xrp0afvBgwe/W9i2Xbt2W+/72qZNm3jnnXfK0vvBbtM6W+rerk3tFt63cCQiLiwY3aEus5mZ2bBhw9bfdNNN+x9zzDHrjj/++HW33XZb+YABAzaUlTV9yLNbt27vderU6b05c+Z0Apg2bdp+TT2noXu4rlixot0DDzzQEWDGjBn7DRs2bIfu7dpUz/VJSV+OiP8qnCjpK8BTO7JiMzPbyezgT2e2x4gRI9bdcMMN3Y899ti3O3fuvKV9+/Zx1FFHNTvYbrnllupzzz23Yq+99tpy7LHHNnlTmfru4ZqeMPXurbfe2vWCCy44qE+fPhu/+tWv1u7I62oqXP8FuDe9DOKz6bQhJMdeT9uRFZuZmY0aNWrd5s2b6/KF6urqF+qGX3nllT8DHHDAART+dnXy5Mlbj4UeffTRG6qqqhbWjV977bWvNra+hu7hWlZWxowZMz4w/amnnqraltdTp6mLSKwGhkk6luQnOQC/8ZWZzMzMGtbkXXEA0jDd5kCVNBK4nuRKTlMj4vv1tDkDmAQEyY3Zz9zW9ZiZmdW55557Ol9xxRU9C6f16tVr4+9///u/Us89XPv3778py6s6QTPDdXtIagPcBJwA1ABPS5oVEQsL2vQF/g04KiLWSmr0B8BmZpapLVu2bFFZWVk03bT1GD169FujR49e2HTL7bdlyxYBWxqav0NXoGjCUGBJRLwUEZuAO4BRRW2+DNwUEWth625oMzNrGS/U1tZ2SYPCmmnLli2qra3tArzQUJvceq5AD6DwN7I1wCeL2vQDkPQYya7jSRHx2+IFSZoITATo3bt3LsWame1uNm/efO7KlSunrly5ciD5drZ2NVuAFzZv3nxuQw3yDNf6vgkV73poC/QFjgF6Ao9IGhgRb7zvSRFTSO4tS2Vl5S61+8LMrFSGDBmyGji11HXsivL8plID9CoY7wkUnyJdA9wXEX+LiJeBKpKwNTMza7XyDNengb6S+khqR3KN4llFbe4FPg0gqRvJbuKXcqzJzMwsd7mFa0RsBi4E5gCLgDsjYoGkyZLqdkPMIblO8ULgD8DX0msXm5mZtVp5HnMlImYDs4umXVkwHMC/pg8zM7Ndgs8OMzMzy5jD1czMLGMOVzMzs4w5XM3MzDLmcDUzM8uYw9XMzCxjDlczM7OMOVzNzMwy5nA1MzPLmMPVzMwsYw5XMzOzjDlczczMMuZwNTMzy5jD1czMLGMOVzMzs4w5XM3MzDLmcDUzM8uYw9XMzCxjDlczM7OMOVzNzMwy5nA1MzPLWK7hKmmkpCpJSyRd3ki70yWFpMo86zEzM2sJbfNasKQ2wE3ACUAN8LSkWRGxsKjd3sDFwJN51WK7r4p3Z5S6hFxVl7oAM6tXnj3XocCSiHgpIjYBdwCj6mn3beAHwLs51mJmZtZi8gzXHsDygvGadNpWkgYDvSLi140tSNJESfMkzautrc2+UjMzswzlGa6qZ1psnSmVAT8CLm1qQRExJSIqI6KyvLw8wxLNzMyyl2e41gC9CsZ7Aq8WjO8NDAQeklQNHAHM8klNZmbW2uUZrk8DfSX1kdQOGAvMqpsZEW9GRLeIqIiICuAJ4NSImJdjTWZmZrnLLVwjYjNwITAHWATcGRELJE2WdGpe6zUzMyu13H6KAxARs4HZRdOubKDtMXnWYmZm1lJ8hSYzM7OMOVzNzMwy5nA1MzPLmMPVzMwsYw5XMzOzjDlczczMMuZwNTMzy5jD1czMLGMOVzMzs4w5XM3MzDLmcDUzM8uYw9XMzCxjDlczM7OMOVzNzMwy5nA1MzPLmMPVzMwsY7neLH1XUPHujFKXkKvqUhdgZrYLcs/VzMwsYw5XMzOzjDlczczMMuZwNTMzy5jD1czMLGO5hqukkZKqJC2RdHk98/9V0kJJz0v6X0kH5VmPmZlZS8gtXCW1AW4CTgQGAOMkDShq9hxQGRGHAXcDP8irHjMzs5aSZ891KLAkIl6KiE3AHcCowgYR8YeI2JCOPgH0zLEeMzOzFpFnuPYAlheM16TTGjIBuL++GZImSponaV5tbW2GJZqZmWUvz3BVPdOi3obSWUAl8MP65kfElIiojIjK8vLyDEs0MzPLXp6XP6wBehWM9wReLW4k6XjgCmBERGzMsR4zM7MWkWfP9Wmgr6Q+ktoBY4FZhQ0kDQZuBk6NiNU51mJmZtZicgvXiNgMXAjMARYBd0bEAkmTJZ2aNvsh0Am4S9J8SbMaWJyZmVmrketdcSJiNjC7aNqVBcPH57l+MzOzUvAVmszMzDLmcDUzM8uYw9XMzCxjDlczM7OMOVzNzMwy5nA1MzPLmMPVzMwsYw5XMzOzjDlczczMMuZwNTMzy5jD1czMLGMOVzMzs4w5XM3MzDLmcDUzM8uYw9XMzCxjDlczM7OMOVzNzMwy5nA1MzPLmMPVzMwsYw5XMzOzjDlczczMMuZwNTMzy1iu4SpppKQqSUskXV7P/PaSfpHOf1JSRZ71mJmZtYTcwlVSG+Am4ERgADBO0oCiZhOAtRFxCPAj4Oq86jEzM2spefZchwJLIuKliNgE3AGMKmozCrgtHb4bOE6ScqzJzMwsd21zXHYPYHnBeA3wyYbaRMRmSW8CXYH
```

Comment:
The `plot_prices()` function is using the marginal of the `COMMODITY_BALANCE` equation - these prices are discounted. You could switch to the auxiliary variable 'PRICE_COMMODITY`, where GAMS does the 'undiscounting' for you.